blobfuse/OAuthTokenCredentialManager.cpp

```diff
@@ -131,19 +146,46 @@ OAuthToken OAuthTokenCredentialManager::get_token()
 }
 
 /// <summary>
-/// TODO: check the expiry time against the current utc time
+/// check the expiry time against the current utc time
 /// <summary>
 bool OAuthTokenCredentialManager::is_token_expired()
 {
     if(!valid_authentication)
+    {
+        syslog(LOG_INFO, "At is_token_expired: valid_authentication is false so token expired");

```

Comment:
Can we update this syslog string to be "At is_token_expired: Token expired" ?

Reply:
done